lib/modules/platform/gitea/utils.ts

```diff
@@ -1,4 +1,13 @@
+import URL from 'url';
+import { PlatformId } from '../../../constants';
+import { CONFIG_GIT_URL_UNAVAILABLE } from '../../../constants/error-messages';
+import { logger } from '../../../logger';
+import type { HostRuleSearchResult } from '../../../types';
+import * as hostRules from '../../../util/host-rules';
 import { regEx } from '../../../util/regex';
+import { parseUrl } from '../../../util/url';
+import type { GitUrlOption } from '../types';
+import type * as helper from './gitea-helper';
```

Comment:
```suggestion
import type { Repo } from './gitea-helper';
```
please import only required type